tests/associations/has-one.test.js

```diff
@@ -67,12 +67,26 @@ describe('Associations(HasOne)', function() {
           postal_code: 'NW1',
           country_code: '44'
         }),
+        test.models.Address.create({
+          street: 'Avenue de l\'Atomium',
+          state_province: 'Brussels',
+          postal_code: '1020',
+          country_code: '32'
+        }),
         test.models.User.create({
           username: 'sherlock',
           email: 'sherlock@holmes.com'
+        }),
+        test.models.User.create({
+          username: 'MannekenPis',
+          email: 'manneken.pis@brussels.be'
         })
-      ]).spread(function(address, user) {
-        return user.setAddress(address);
+      ]).spread(function(address, address2, user, user2) {
+        return user.setAddress(address).then(function(){
+          user2.setAddress(address2).then(function(){
```

Comment:
I do that for fix the error on node v0.10: https://github.com/Fridus/epilogue/commit/50142a483f46dd14741f7e9fdd18dc34bd407067
https://travis-ci.org/dchester/epilogue/builds/64384659

This works on my desktop:
```
... .spread(function(address, address2, user, user2) {
        user.setAddress(address);
        return user2.setAddress(address2);
      });
```